deploy/dependencies/korrel8r/korrel8r_svc.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
```

Comment:
Need annotation to request serving certs from OCP ,see:
https://github.com/korrel8r/operator/blob/main/controllers/korrel8r_controller.go#L343